File: Space_boot/src/main/java/space/rest/request/StartRequest.java
package space.rest.request;

public class StartRequest {

    private Integer UserId;
    private Integer idEspece;

    public Integer getUserId() {
        return UserId;
    }

    public void setUserId(Integer userId) {
        UserId = userId;
    }

    public Integer getIdEspece() {
        return idEspece;
    }

    public void setIdEspece(Integer idEspece) {
        this.idEspece = idEspece;
    }
}

File: Space_front/src/app/class/request/start-request.ts
export class StartRequest {
        constructor(
        public userId: number,
        public idEspece: number,
    ) { }

    public getIdEspece(): number {
        return this.idEspece;
    }
    public getUserId(): number {
        return this.userId;
    }
    public setIdEspece(idEspece: number): void {
        this.idEspece = idEspece;
    }
    public setUserId(userId: number): void {
        this.userId = userId;
    }
}
